spec/rails/doc/smart/expected.yaml

```diff
@@ -20,40 +20,22 @@ paths:
       tags:
       - Table
       parameters:
-      - name: page
-        in: query
-        schema:
-          type: integer
-        example: 42
-      - name: per
-        in: query
-        schema:
-          type: integer
-        example: 10
       - name: X-Authorization-Token
         in: header
         required: true
         schema:
           type: string
         example: token
-      - name: filter[name]
```

Comment:
Note) `filter[name]` was removed because it matched the pattern `3. paths.*.*.parameters`